Server/docker-compose.yaml

```diff
@@ -20,32 +20,39 @@ version: "3.7"
 services:
 
   settings-tp:
-    image: hyperledger/sawtooth-settings-tp:1.0
+    image: hyperledger/sawtooth-settings-tp:1.2
     container_name: sawtooth-settings-tp-testnet
     depends_on:
       - validator
-    expose:
-      - 4004
-    command: settings-tp -C tcp://validator:4004
+    entrypoint: settings-tp -C tcp://validator:4004
     stop_signal: SIGKILL
+    logging:
+        driver: "json-file"
+        options:
+            max-file: "5"
+            max-size: "10m"
 
   validator:
-    image: gluwa/creditcoin-validator:1.7
+    image: gluwa/creditcoin-validator-priv:testnet-latest
```

Comment:
Don't we want to adopt some numbering scheme? E.g. -rc01, 02 etc ? Makes it much easier when switching between containers in case of issues.

Reply:
I think we'll have to notify them anyway so having a versioned name sounds better to me. I can bump this as soon as version start to be released.